src/uct/ugni/base/ugni_pd.c

```diff
@@ -337,7 +337,6 @@ UCT_PD_COMPONENT_DEFINE(uct_ugni_pd_component,
                         uct_ugni_query_pd_resources,
                         uct_ugni_pd_open,
                         NULL,
-                        (3 * sizeof(uint64_t)),
```

Comment:
it's already present there. in uct_ugni_pd_query()  :
pd_attr->rkey_packed_size  = 3 * sizeof(uint64_t);